### PR TITLE
For #406: bump java to version 11

### PR DESCRIPTION
**Pull request by paulodamaso**
For #406: Bump java to version 11 to use new rultor image

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#426](https://github.com/yegor256/jpeek/issues/426) is assigned to @fanifieiev[/z](https://www.0crat.com/u/fanifieiev), here is [why](https://www.0crat.com/footprint/C7JGJ00DP/89fbf5f1-a17b-4ad8-ba02-10f3721ed87b); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: src/main/java/org/jpeek/web/Reports.java [hunk 1]
```diff
@@ -164,10 +165,14 @@ private static void extractClasses(final Path path) throws IOException {
      */
     private static void deleteIfPresent(final Path dir) throws IOException {
         if (Files.exists(dir)) {
-            Files.walk(dir)
-                .sorted(Comparator.reverseOrder())
-                .map(Path::toFile)
-                .forEach(File::delete);
+            try (
```
Comment (fanifieiev) [inline src/main/java/org/jpeek/web/Reports.java]: @paulodamaso 
I think the following is a better Cactoos approach:
```
new ForEach<Path>(
            new ProcOf<>(
                path -> path.toFile().delete()
            )
        ).exec(
            new Sorted<>(
                Comparator.reverseOrder(),
                new Directory(dir)
            )
        );       
```

File: pom.xml [hunk 2]
```diff
@@ -268,8 +268,8 @@ SOFTWARE.
           <artifactId>maven-compiler-plugin</artifactId>
           <version>3.8.0</version>
           <configuration>
-            <source>1.8</source>
-            <target>1.8</target>
+            <source>11</source>
```
Comment (fanifieiev) [inline pom.xml]: @paulodamaso What about moving the versions to 
`<properties>...</properties>`
section in pom.xml ?

File: pom.xml [hunk 3]
```diff
@@ -284,7 +284,7 @@ SOFTWARE.
           <plugin>
             <groupId>com.qulice</groupId>
             <artifactId>qulice-maven-plugin</artifactId>
-            <version>0.18.17</version>
+            <version>0.18.19</version>
```
Comment (fanifieiev) [inline pom.xml]: @paulodamaso Same as above

**Comment by 0crat** [comment]
@fanifieiev[/z](https://www.0crat.com/u/fanifieiev) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by paulodamaso** [comment]
@fanifieiev This issue is outdated

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @fanifieiev[/z](https://www.0crat.com/u/fanifieiev), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
Pull request [#426](https://github.com/yegor256/jpeek/issues/426) was not merged, no payment for `ARC`, see [§28](http://www.zerocracy.com/policy.html#28)

**Comment by sereshqua** [comment]
@paulodamaso even though the issue is outdated, i need to give QA verdict: 

please make sure that all problems found by the code reviewer (@fanifieiev this time) would be next time addressed by the pull request author (you)

**Comment by paulodamaso** [comment]
@sereshqua I will try to address all problems in the next pull requests, thank you

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Pull request closed without merging**